### DISCUSSION TITLE
Localization

### DISCUSSION THREAD

**Discussion by orangesunny** [simple-login/app#390] [Kind words ❤️]
Hi, I discovered your lovely project a week ago. Going to buy a subscription and use it on my domain. Thanks!

As I do [Weblate](https://github.com/WeblateOrg/weblate) I want all great libre projects to be localized and closer to their users. So I am offering my help with the localization effort of the SimpleLogin app, website, etc. We at Weblate offer free hosting for libre projects like SimpleLogin is 😃 
As I am not a developer, I want to contribute this way.

**Comment by nguyenkims** [comment]
Thanks, weblate looks great and is also recommended by other users :). 

We are trying out weblate and https://poeditor.com/ and haven't decided yet. Weblate seems better suited for open source projects like SimpleLogin and poeditor has some issues with importing/exporting our localization files so we are going to use weblate probably :).
